wcomponents-theme/src/main/js/wc/ui/draggable.js

```diff
@@ -295,16 +295,17 @@ function(attribute, clearSelection, event, getMouseEventOffset, isAcceptableEven
 		 *
 		 * @function
 		 * @private
-		 * @param {Element} element the element being shown.
-		 * @param {String} action The shed action SHOW or HIDE.
+		 * @param {Event} $event the shed event that fired.
 		 */
-		function shedAjaxSubscriber(element, action) {
+		function shedAjaxSubscriber($event) {
+			var element = $event.target,
```

Comment:
Pretty sure this breaks `shedAjaxSubscriber` when it is called via `processResponse.subscribe` - looks like the introduction of a bug to me.

Reply:
This is definitely a problem - but why didn't our testing pick it up (well the answer to that is obvious and so I need to go write all those tests I have been promising since we moved off Doh) ?